modules/table/src/main/java/org/apache/ignite/internal/table/distributed/index/IndexBuildTaskId.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.index;
+
+import java.util.UUID;
+import org.apache.ignite.internal.tostring.S;
+
+/**
+ * {@link IndexBuildTask} ID.
+ */
+class IndexBuildTaskId {
+    private final UUID tableId;
+
+    private final int partitionId;
+
+    private final UUID indexId;
+
+    IndexBuildTaskId(UUID tableId, int partitionId, UUID indexId) {
+        this.tableId = tableId;
+        this.partitionId = partitionId;
+        this.indexId = indexId;
+    }
+
+    public UUID getTableId() {
```

Comment:
we don't use `get` prefixes

Reply:
This rule is valid for 2.0 in 3.0 we don't have it.